code/modules/client/preferences.dm

```diff
@@ -1375,7 +1375,7 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 					if(unlock_content)
 						toggles ^= MEMBER_PUBLIC
 				if("gender")
-					var/chosengender = input(user, "Select your character's gender.", "Gender Selection", gender) in list(MALE,FEMALE,"nonbinary","object")
+					var/chosengender = input(user, "Select your character's gender.", "Gender Selection", gender) in list(MALE,FEMALE)
```

Comment:
I think this PR is taking commits from your others

Reply:
Looks like you already fixed this.

Let the people identify as toasters, I say.